### PR TITLE
Update Readme for 2.4 related stuff

### PR DESCRIPTION
**Pull request by jackjt8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ commands run with `sudo` will be affected.
 </details>
 
 More information about Chunky, including a short getting started guide and
-rendering tips are available at the [Chunky Documentation page][1]. For more insights into Chunky's development, see the [development blog][4].
+rendering tips are available at the [Chunky Documentation page][1]. For more insights into Chunky's development, keep an eye on the Discord; messages from Contributors can sometimes give you insite into what everyone is working on.
```
Comment (leMaik) [inline README.md:138]:
```suggestion
rendering tips are available at the [Chunky Documentation page][1]. For more insights into Chunky's development, keep an eye on the Discord; messages from contributors can sometimes give you insight into what everyone is working on.

```

File: README.md [hunk 2]
```diff
@@ -186,18 +186,19 @@ See the file `licenses/commons-math.txt` for the copyright notices.
...
 See the file `licenses/fast-util.txt` for the copyright notice.
 
-[1]: http://chunky.llbit.se
-[2]: http://chunky.llbit.se/troubleshooting.html
+[1]: https://chunky-dev.github.io/docs/
+[2]: https://chunky-dev.github.io/docs/faq/troubleshooting/
 [3]: http://www.reddit.com/r/chunky
-[4]: http://llbit.se
-[5]: http://chunky.llbit.se/headless.html
+
```
Comment (leMaik) [inline README.md:192]: Maybe we should stop using numbers and use IDs instead, like `[chunky-docs]: https://chunky-dev.github.io/docs`?

**Review by leMaik — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jackjt8) [reply README.md:193]: That might be easier and more editor friendly. I'll work on that shortly

Comment (jackjt8) [reply README.md]: Addressed in a0388c8

**Comment by leMaik** [comment]
Thank you! :raised_hands: I always forget the readme when updating chunky…

**Pull request merged**